src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/ClusterAwareConnectionProxy.java

```diff
@@ -723,7 +729,7 @@ private synchronized void connectTo(int hostIndex) throws SQLException {
                 new StringBuilder("Connection to ")
                         .append(isWriterHostIndex(hostIndex) ? "writer" : "reader")
                         .append(" host '")
-                        .append(host == null ? "<null>" : host.getHostPortPair())
+                        .append(host.getHostPortPair())
```

Comment:
No harm to leave a safer statement.

Reply:
The only way the host would be null at this point is if we had a null entry in the topology list, which won't happen as far as I can tell since we removed the null writer implementation. However if you think I should add it back still I can do that

Reply:
I see your point. I just prefer to be on safe side.